src/pages/Dashboard/OverviewCard.tsx

```diff
@@ -283,6 +282,19 @@ export function OverviewCard({
     [v1ArbitrumWeeklyFees, v1AvalancheWeeklyFees, v2ArbitrumWeeklyFees, v2AvalancheWeeklyFees]
   );
 
+  const formattedDuration = useMemo(() => {
+    const now = Date.now() / 1000;
+    const days = Math.floor((now - feesSummary.lastUpdatedAt) / (3600 * 24));
+    let restHours = Math.round((now - feesSummary.lastUpdatedAt) / 3600) - days * 24;
+    if (days === 0) {
+      restHours = Math.max(restHours, 1);
+    }
+
+    const daysStr = days > 0 ? `${days}d` : "";
+    const hoursStr = restHours > 0 ? `${restHours}h` : "";
+    return [daysStr, hoursStr].filter(Boolean).join(" ");
```

Comment:
What if both are `""`?

Reply:
if day = 0 then we force hours to be 1. so i think case when both are "" is not possible